src/prpy/util.py

```diff
@@ -897,6 +979,37 @@ def IsTimedTrajectory(trajectory):
     empty_waypoint = numpy.zeros(cspec.GetDOF())
     return cspec.ExtractDeltaTime(empty_waypoint) is not None
 
+
+def IsJointSpaceTrajectory(traj):
+    """
+    Check if trajectory is a joint space trajectory.
+
+    @param openravepy.Trajectory traj: A path or trajectory.
+    @return bool result: Returns True or False.
+    """
+    try:
+        if traj.GetConfigurationSpecification().GetGroupFromName("joint_values"):
+            return True
+    except openravepy.openrave_exception:
+        pass
+    return False
+
+
+def IsTrajectoryTypeIkParameterization(traj):
+    """
+    Check if trajectory is a workspace trajectory.
+
+    @param openravepy.Trajectory traj: A path or trajectory.
+    @return bool result: Returns True or False.
+    """
+    try:
+        if traj.GetConfigurationSpecification().GetGroupFromName("ikparam_values"):
```

Comment:
You should also check the *type* of the `IkParameterization` here. I believe this is as simple as appending the name of the parameterization to the string you pass to `GetGroupFromName`.